README.md

```diff
@@ -1,3 +1,43 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+#README
+
+	###Check-Ins
+
+  We had async check-ins in the morning including status updates, with a requested response by 9 AM MT. If working in the evening, Slack status updates before signing off for the next morning are acceptable.
+Live group check-ins during class group time. Stand up responses state the status of assigned tasks; what the programmer will be working on that day; and questions, comments and concerns that inform the group.
+
+  ###Workflow
```

Comment:
Space between # and word

Reply:
The indentation on lines 9 and 10 looks off in the editor. Can you please adjust?

Reply:
Corrected space and indention